devtools/build/tools/wolfi-apk-extract/main.go

```
@@ -0,0 +1,189 @@
+	return tw.Close()
+}
+
+func run(in, out string, allow []string) error {
+	if len(allow) == 0 {
+		return errors.New("--keep is required (at least one path prefix)")
+	}
+	f, err := os.Open(in)
+	if err != nil {
+		return fmt.Errorf("open input: %w", err)
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)